src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -75,19 +95,7 @@ function ExportAnnotations({
       {exportableAnnotations.length > 0 ? (
         <>
           <label data-testid="export-count" htmlFor="export-filename">
-            Export{' '}
-            <strong>
-              {nonReplyCount}{' '}
-              {pluralize(nonReplyCount, 'annotation', 'annotations')}
-            </strong>{' '}
-            {replyCount > 0
-              ? `(and ${replyCount} ${pluralize(
-                  replyCount,
-                  'reply',
-                  'replies',
-                )}) `
-              : ''}
-            in a file named:
+            Name of export file:
```

Comment:
I have noted we should recover this behavior on a follow-up PR. Not sure in what form, maybe we can make the dropdown display entries like this:

* `Joh Doe (34)`
* `Jane Doe (56, 10 replies)`
* ...

Or something in those lines.

Reply:
Yes. Maybe `Jane Doe (56 annotations, 10 replies)` or `Jane Doe (56 + 10 replies)`. Check in Slack and see what people think.